### DISCUSSION TITLE
A couple of ideas

### DISCUSSION THREAD

**Discussion by diverswan9** [nagadomi/nunif#167] [Ideas]
This is really two separate things. One idea (request) is to have a way of saving a  'preset' of all the settings. I find myself using a variety of settings depending on the video/photo type.

The other is it possible to locate and edit the file for "add metadata to filename". I would like to simplify it. Most of the settings it displays are always the same. I do like having it though. It comes in handy.

Thanks! Much mojo to you...

**Comment by nagadomi** [comment]
About preset, I also think it is better to have it.
But I have been thinking that for a year and still not implemented it.

**Reply by diverswan9** [reply]
What about the other part?  "Add Metadata to Filename"  Is that editable or would you prefer us not to tinker with it? I would assume it is a particular file that could perhaps be edited.

**Reply by nagadomi** [reply]
I am not sure what that part means.
Do you mean that you want to choose which settings to add to the filename as metadata?

**Reply by diverswan9** [reply]
Yes. There are some settings that are always the same for me so I don't really need them every time. I just thought maybe there was a file lurking somewhere in the depths of the "nunif-windows" folder that I could edit myself. This is really not a very 'important' topic I know. I just thought I'd ask.
